client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -5,11 +5,21 @@ import isAtomicSite from 'calypso/state/selectors/is-site-wpcom-atomic';
 import siteHasFeature from 'calypso/state/selectors/site-has-feature';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import {
+	STATS_SUBTYPE_DATE_CONTROL,
+	STATS_SUBTYPE_DOWNLOAD_CSV,
+	STAT_TYPE_STATS_CLICKS,
+	STAT_TYPE_STATS_REFERRERS,
+	STAT_TYPE_STATS_SEARCH_TERMS,
+} from '../constants';
 
-// TODO: define shared variables
-const trafficPaidStats = [ 'statsSearchTerms', 'statsClicks', 'statsReferrers' ];
+const paidStats = [
+	STAT_TYPE_STATS_SEARCH_TERMS,
+	STAT_TYPE_STATS_CLICKS,
+	STAT_TYPE_STATS_REFERRERS,
+];
 
-const featureFlags = [ 'stats/date-control', 'download-csv' ];
+const granularControlForPaidStats = [ STATS_SUBTYPE_DATE_CONTROL, STATS_SUBTYPE_DOWNLOAD_CSV ];
```

Comment:
Will this `stats/date-control` change also affect [the site.jsx](https://github.com/Automattic/wp-calypso/blob/try/paid-stats-naming/client/my-sites/stats/site.jsx#L219)?

Reply:
Nope, that's a different thing, I was using it for a name and wanted to change it later (in this PR) to not cause confusion.